core/components/molecules/tabs/__stories__/InlineContent.story.jsx

```diff
@@ -20,25 +20,29 @@ export const inlineContent = () => {
   };
 
   return (
-    <Tabs onTabChange={onTabChangeHandler} className="mb-6">
-      <Tab label="All" count={12}>
-        <div>All</div>
-      </Tab>
-      <Tab label="Active" count={3}>
-        <div>Active</div>
-      </Tab>
-      <Tab label="Inactive" count={9}>
-        <div>Inactive</div>
-      </Tab>
-      <div className="d-flex justify-content-end flex-grow-1">
-        <div style={{ width: 'var(--spacing-9)' }} className="ml-8">
-          <Input placeholder="Search by name" icon="search" />
+    <Row>
+      <Tabs onTabChange={onTabChangeHandler} className="mb-6">
+        <Tab label="All" count={12}>
+          <div>All</div>
+        </Tab>
+        <Tab label="Active" count={3}>
+          <div>Active</div>
+        </Tab>
+        <Tab label="Inactive" count={9}>
+          <div>Inactive</div>
+        </Tab>
+        <div className="d-flex justify-content-end flex-grow-1 align-items-center">
+          <div>
+            <Input placeholder="Search by name" icon="search" />
+          </div>
+          <Column size={3}>
+            <div className="ml-4">
```

Comment:
Please wrap this inside the `Row` component

Reply:
this does not work

Reply:
Please nest Row and Columns properly.